src/components/Database/DatabasePicker.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from 'react';
+import { connect } from 'react-redux';
+import { AppState } from '../../store';
+import {
+  List,
+  ListGroup,
+  ListGroupSubheader,
+  ListItem,
+  ListItemGraphic,
+  ListItemText,
+  ListDivider,
+} from '@rmwc/list';
+import { Typography } from '@rmwc/typography';
+import '@rmwc/icon/icon.css';
+import './DatabasePicker.scss';
+
+export interface PropsFromState {
+  databases?: string[];
+}
+
+export type Props = PropsFromState & {
+  primary: string;
+  current: string;
+  navigation: (db: string) => ReactElement;
+};
+
+const databaseIcon = (
```

Comment:
Do we want this anywhere else? Consider pulling out into /icons.ts?

Reply:
Agreed, static asset, make it's own file